modules/swagger-project-jakarta/modules/swagger-maven-plugin-jakarta/src/main/java/io/swagger/v3/plugin/maven/jakarta/JakartaTransformer.java

```diff
@@ -131,61 +132,55 @@
      * Extracts Jar into temp directory
      *
      */
-    private static void extract(String inPath, String outPath) throws Exception {
-
-        Path target = Paths.get(outPath);
-        Path source = Paths.get(inPath);
-
-        try (ZipInputStream zis = new ZipInputStream(new FileInputStream(source.toFile()))) {
-
-            // list files in zip
-            ZipEntry zipEntry = zis.getNextEntry();
-
-            while (zipEntry != null) {
-
-                boolean isDirectory = false;
-
-                if (zipEntry.getName().endsWith(File.separator)) {
-                    isDirectory = true;
-                }
-
-                Path targetDirResolved = target.resolve(zipEntry.getName());
-
-                Path normalizePath = targetDirResolved.normalize();
-                if (!normalizePath.startsWith(target)) {
-                    throw new IOException("Bad zip entry: " + zipEntry.getName());
-                }
-
-                Path newPath = normalizePath;
-
-                if (isDirectory) {
-                    Files.createDirectories(newPath);
-                } else {
-                    if (newPath.getParent() != null) {
-                        if (Files.notExists(newPath.getParent())) {
-                            Files.createDirectories(newPath.getParent());
-                        }
-                    }
-                    Files.copy(zis, newPath, StandardCopyOption.REPLACE_EXISTING);
-                }
-
-                zipEntry = zis.getNextEntry();
-
-            }
-            zis.closeEntry();
-
-        }
-    }
+	private static void extract(String inPath, String outPath) throws Exception {
```

Comment:
Would use spaces instead of tabs..

Reply:
You are right, fixed it. In my defense the formatters in the [Coding-Standards](https://github.com/swagger-api/swagger-core/wiki/Coding-Standards) give me 404s.